moveit_experimental/moveit_jog_arm/src/jog_interface_base.cpp

```diff
@@ -105,6 +104,23 @@ bool JogInterfaceBase::readParameters(ros::NodeHandle& n)
   error += !rosparam_shortcuts::get("", n, parameter_ns + "/publish_joint_accelerations",
                                     ros_parameters_.publish_joint_accelerations);
 
+  // This parameter name was changed recently. Give a deprecation warning.
+  if (n.hasParam(parameter_ns + "/status_topic"))
+  {
+    error += !rosparam_shortcuts::get("", n, parameter_ns + "/status_topic", ros_parameters_.status_topic);
+  }
+  else if (n.hasParam(parameter_ns + "/warning_topic"))
+  {
+    error += !rosparam_shortcuts::get("", n, parameter_ns + "/warning_topic", ros_parameters_.status_topic);
+    ROS_WARN_NAMED(LOGNAME,
+                   "Topic 'warning_topic' was renamed to 'status_topic'. Please update the jogging yaml file.");
+  }
+  else
+  {
+    ROS_WARN_NAMED(LOGNAME,
+                   "Topic 'warning_topic' was renamed to 'status_topic'. Please update the jogging yaml file.");
+  }
```

Comment:
I don't think you need to test if warning_topic is there, just print the log and call get, if neither are there you want an error.

Reply:
That makes sense. Done